apps/wallet/.prettierignore

```diff
@@ -0,0 +1,9 @@
+dist/
```

Comment:
had to restore the directory level .prettierignore files because prettier doesn't scan parent directories for ignore files